docs/src/main/paradox/discovery.md

```diff
@@ -176,11 +176,55 @@ spec:
           protocol: TCP
 ```
 
+## Discovery Method: AWS EC2 Discovery
+
+If you're an AWS user, you can use tags to simply mark the instances that belong to the same cluster. Use a tag that
+has "service" as the key and set the value to be the name of your service. Note that this implementation is adequate
+for users running service clusters on *vanilla* EC2 instances. If you're using Amazon EKS (Amazon Elastic Container 
+Service for Kubernetes) or plain Amazon ECS, then this is not the right choice and you may want to use 
+the @ref:['Kubernetes API'-based discovery method](discovery.md#discovery-method-kubernetes-api) or some other solution.
+
```

Comment:
For reference, auto-scaling groups can apply a tag to every new instance, which should handle propagation nicely. Not sure if it's worth calling all that out here or not.

Reply:
@easel I think it's worth calling out since auto-scaling is something akka users generally want :-) You want to PR ? cheers!

Reply:
Ok, will do once this lands, there's no good way to pr against an open pr is there?

Reply:
It's possible actually, one has to go to the persons repo and target that branch that is this PR. But anwyay, this will shortly be in :)

Reply:
@ktoso i think the build server says ok now, so this can be merged ? cheers!

Reply:
@easel this is merged now so feel free to PR